dell/src/main/java/org/apache/iceberg/dell/DellProperties.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.dell;
+
+import java.io.Serializable;
+import java.util.Map;
+
+public class DellProperties implements Serializable {
+  /**
+   * S3 Access key id of Dell EMC ECS
+   */
+  public static final String ECS_S3_ACCESS_KEY_ID = "ecs.s3.access-key-id";
+
+  /**
+   * S3 Secret access key of Dell EMC ECS
+   */
+  public static final String ECS_S3_SECRET_ACCESS_KEY = "ecs.s3.secret-access-key";
```

Comment:
In dell ecs, we name the secret key as `secret-access-key` ? In our alibaba oss, we name it as `access-key-secret`. https://www.alibabacloud.com/help/en/doc-detail/29009.htm 

Well,  I see the aws also name it as `secret-access-key`, it is okay to follow it for dell ecs. https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html

Reply:
We re-implement the AWS S3 protocol, so the configurations have the same name.